pallets/loans/src/lib.rs

```diff
@@ -446,6 +446,11 @@ pub mod pallet {
                     }
                 }
             });
+
+            // This is used to trigger the price aggregation to update the results to the ORML Oracle Pallet.
+            for currency_id in Currencies::<T>::get().iter() {
+                let _ = Self::get_price(currency_id);
```

Comment:
This needs to be better weighted.

Reply:
Is any suggestion here?

Reply:
The `on_initialized` function should be able to return the weight.

Reply:
We moved the code into `on_finalize` in #219 and maybe needn't anymore.